deploy/crds/smartgateway.infra.watch_smartgateways_crd.yaml

```diff
@@ -47,6 +47,9 @@ spec:
               logLevel:
                 description: Log level for sg-core.
                 type: string
+              blockEventBus:
```

Comment:
I think these new options should be added to the example CR in this directory, and they also need a reference in _/deploy/olm-catalog/smart-gateway-operator/manifests/smartgateway.infra.watch_smartgateways_crd.yaml_ as well, though to be honest I don't recall if that one is auto-generated during some OLM build step

Reply:
right, I forgot about that. I think there is a command for generating this in operator-sdk.

Reply:
Yes, you just adjust the files in `deploy/crds/` and then you run this:

```
operator-sdk bundle generate --channels unstable --default-channel unstable
```

Once you run that everything you change in `deploy/crds/` will be sync'd to the `deploy/olm-catalog/` locations.